analytic_tag_dimension/readme/CONTRIBUTORS.rst

```diff
@@ -2,6 +2,6 @@
 * Artem Kostyuk <a.kostyuk@mobilunity.com>
 * Jaume Planas <jaume.planas@minorisa.net>
 * `Tecnativa <https://www.tecnativa.com>`__:
-
```

Comment:
Don't remove this empty line that is needed for proper RST syntax

Reply:
Fixed. Empty line added.